test/extract.spec.ts

```diff
@@ -209,6 +209,12 @@ describe('extractResult()', function () {
                     value: 40537.456,
                     extra: '30001 times',
                 },
+                {
```

Comment:
This is the backward compatible metric as before v1.18.0 it was resulting like that in case of multiple metrics